pyblish_clarisse/app.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# coding=utf-8
+'''
+Author: zuokangbo
```

Comment:
These are not necessary, as the information will be in the commit history. It can also discourage others from contributing, since it looks like there can be only one "owner". As soon as someone does contribute, the information is also inaccurate as there will be more than one author.

Can we remove these?

Reply:
Ok